cmd/titus-standalone/pod.jsonnet

```diff
@@ -0,0 +1,140 @@
+{
+  "metadata": {
```

Comment:
What is your thinking behind leaving this here.
Don't you think it will just be stale and not work?

Reply:
The idea was to give someone a thing to work from. What do you think?

Reply:
I think we should remove all the things we can, strip it to bare bones